### DISCUSSION TITLE
Location of downloaded models

### DISCUSSION THREAD

**Discussion by MudiLoodi** [idiap/coqui-ai-TTS#211] [Q&A] [answered]
I want to delete some unused models, but I can't manage to find the installation location. I'm talking about the models that get automatically installed when using for the first time.

**Comment by eginhard** [comment, marked as answer]
```python
from trainer.io import get_user_data_dir
print(get_user_data_dir("tts"))
```

**Reply by MudiLoodi** [reply]
It prints `username\AppData\Local\tts` but this directory does not exists.

**Reply by eginhard** [reply]
Strange, that should work. Can you run the following and see what it prints:
```python
from TTS.api import TTS
api = TTS()
print(api.manager.download_model("tts_models/multilingual/multi-dataset/xtts_v2"))
```
The first part will be the location of the model. You can use any other model name if you have any specific one downloaded before.

**Reply by MudiLoodi** [reply]
The first part is `username\\AppData\\Local\\tts\\tts_models--en--jenny--jenny\\model.pth`. Still points to the non-existent `tts` folder. Pretty weird. My C drive is almost full I need to delete some models😆

**Reply by eginhard** [reply]
If that code ran successfully, the folder will exist, you're probably not looking in the right place. The full path should probably be `C:\Users\<user>\AppData\Local\tts\`

**Reply by MudiLoodi** [reply]
Well I checked many times, I'm looking in the same path you provided, it is still not there. Will the models get uninstalled if I uninstall the package with pip?

**Reply by eginhard** [reply]
No, the models don't get removed. If `download_model()` ran successfully, it must be somewhere. You can try searching for the `tts_models--en--jenny--jenny` folder with the Windows search function.

**Reply by MudiLoodi** [reply]
Finally found it! The models are stored in
`C:\Users\<username>\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\Local\tts`

Maybe this was a unique case.
